### PR TITLE
Issue678 cleanup

### LINKED ISSUE #678: various clean ups
This branch is to clean up various file formatting issues.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #678 
Adds documentation and does various file cleanups.
The regression tests work without error.

### CODE REVIEW

File: Annex60/Fluid/HeatExchangers/Validation/ConstantEffectiveness.mo [hunk 1]
```diff
@@ -3,51 +3,64 @@ model ConstantEffectiveness
...
+    "Boundary condition"
+    annotation (Placement(transformation(extent={{-58,-10},
             {-38,10}})));
     Modelica.Blocks.Sources.Ramp PIn(
     height=200,
     duration=60,
     offset=101325,
     startTime=50)
-                 annotation (Placement(transformation(extent={{-20,-50},{0,-30}})));
+    "Ramp signal for pressue"
```
Comment (Mathadon) [inline Annex60/Fluid/HeatExchangers/Validation/ConstantEffectiveness.mo]: typo

File: Annex60/Fluid/HeatPumps/package.mo [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-</p>
-</html>"));
-
-end HeatPumps;
+within Annex60.Fluid;
+package HeatPumps
+  extends Modelica.Icons.VariantsPackage;
+annotation (preferredView="info", Documentation(info="<html>
+<p>
+This package contains components models for heat pumps.
```
Comment (Mathadon) [inline Annex60/Fluid/HeatPumps/package.mo]: typo: componentS models

**Comment by mwetter** [comment]
@Mathadon Thanks for catching, there where actually many of the 2nd typos...
